go.mod

```diff
@@ -103,7 +103,7 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e
 	google.golang.org/genproto v0.0.0-20200806141610-86f49bd18e98
-	google.golang.org/grpc v1.31.0
+	google.golang.org/grpc v1.33.0-dev.0.20200828165940-d8ef479ab79a
```

Comment:
Updates grpc to 1.33.0-dev i.e. to the latest commit. By the time we release 1.8, we can go back to 1.33 main release I guess. LMK, if thee are any concerns with this.

Reply:
I was thinking the same, so SGTM. I have alerts for gRPC releases so we shouldn't miss the 1.33 release